launcher/constants.py

```diff
@@ -162,22 +195,42 @@
       - kiwi_live_image_test:
           description: {KIWI_TEST_SUITES['kiwi_live_image_test']['description']} for {distri} {version}
 
+    kiwi-{distri}-{version}-live-iso-x86_64-efi:
+      - kiwi_live_image_test_efi:
+          description: {KIWI_TEST_SUITES['kiwi_live_image_test_efi']['description']} for {distri} {version}
+
     kiwi-{distri}-{version}-install-iso-x86_64:
       - kiwi_live_image_test:
           description: {KIWI_TEST_SUITES['kiwi_live_image_test']['description']} for {distri} {version}
           settings:
-            PUBLISH_HDD_1: "%DISTRI%-%VERSION%-%PACKAGE%-%ARCH%-%BUILD%.qcow2"
+            PUBLISH_HDD_1: "kiwi-%DISTRI%-%VERSION%-%PACKAGE%-%ARCH%-%BUILD%.qcow2"
```

Comment:
So I saw two types , one is ending with `-efi` and others don't have any ending , so I assume the one with no ending is actually `no-efi` . WDYT to make it explicit in both cases ?
```suggestion
            PUBLISH_HDD_1: "kiwi-%DISTRI%-%VERSION%-%PACKAGE%-%ARCH%-%BUILD%-no-efi.qcow2"
```

Reply:
Hm, it would be a bit clearer indeed. However, no one else on o3 appears to follow that convention. But I guess nothing bad will happen if we start using it :thinking:

Reply:
Some test cases also add @%MACHINE% and custom string, e.g. `SLES-%VERSION%-%ARCH%-%BUILD%@%MACHINE%-minimal_with_sdk%BUILD_SDK%_installed.qcow2` (create_hdd_minimal_base+sdk in o.s.d)